### ISSUE TITLE
Align project work with existing data strategy efforts

### ISSUE THREAD

**Issue by mheadd** [18F/data-federation-project#81]
## User story
As an agency data lead, I want understand how this project fits in with the broader federal data strategy so that I can benefit from participating.
  
## Acceptance criteria
- [x] Meet with OMB and agency leads involved in federal data strategy efforts.
- [x] Develop overview / introductory materials for Federal Data Strategy working groups.
- [x] Incorporate connection into promotional materials (#77).

**Comment by juliaklindpaintner** [comment]
Opened a related issue (#90) in response to confusion from colleagues around the relationship between data.gov and US Data Federation.

**Comment by juliaklindpaintner** [comment]
I want to be able to articulate an answer to the following questions: 
- [ ] How does the US Data Federation relate to data.gov?
- [ ] How does the US Data Federation relate to the Federal Data Strategy?

**Comment by juliaklindpaintner** [comment]
TL; DR: They are all evidence of a growing understanding of data management as crucial infrastructure for the 21st century. 
- data.gov is a platform for the US government's open data
- the Federal Data Strategy is a cross-agency initiative to develop more consistent practices around data management
- the US Data Federation is a 10x project developing resources to support federated data efforts.

**How does US Data Federation relate to data.gov?**
Data.gov is the home of the US government's open data. Federal agencies use data.gov is a platform for government agencies to make their data available to the public. 
The goal of the US Data Federation is to support efforts in which data—whether open and not—is collected across organizational boundaries (e.g., a federal agency collecting data from states). The US Data Federation is doing that by curating resources and developing reusable tools and repeatable processes to make the creation of high-quality datasets easier. 

**How does the US Data Federation relate to the Federal Data Strategy?**
The Federal Data Strategy is an cross-agency initiative to develop a more consistent​ ​approach to federal data stewardship, use,​ ​and access, in order to meet the Cross-Agency Priority (CAP) Goal: Leveraging Data as a Strategic Asset (part of the President's Management Agenda).
